src/labels/CustomLabels.labels

```diff
@@ -6718,7 +6790,303 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Import Data in Get Started</shortDescription>
+        <value>Import Data</value>
+    </labels>
+     <labels>
+        <fullName>gsChecklistItemImportDataLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist item Import Data in Get Started</shortDescription>
+        <value>Learn more about importing data in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemImportDataSecBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Import Data in Get Started</shortDescription>
+        <value>See Documentation</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemImportDataTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Import Data in Get Started</shortDescription>
+        <value>Import Data</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemManageChangeDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist item Manage Change in Get Started</shortDescription>
+        <value>Salesforce is updated frequently to address issues and add new features. Understanding these changes and how to manage the update process will ensure your organization is always ready to take full advantage of the platform.</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemManageChangeLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link Label of checklist item Manage Change in Get Started</shortDescription>
+        <value>Learn more about Release Readiness in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemManageChangePriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of checklist item Manage Change in Get Started</shortDescription>
+        <value>View Release Readiness Group</value>
+    </labels>
+    <labels>
+        <fullName>gsChecklistItemManageChangeTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of checklist item Manage Change in Get Started</shortDescription>
+        <value>Manage Changes to your Instance</value>
```

Comment:
@fparodi-salesforce Can you please change instance to "Org"? We shouldn't use the word "instance."